docs/content/installation/integrations/app-protect-waf-v5/installation.md

```diff
@@ -0,0 +1,241 @@
+---
+docs: DOCS-000
+doctypes:
+- ''
+title: Build NGINX Ingress Controller with NGINX App Protect WAF v5
+toc: true
+weight: 100
+---
+
+This document explains how to build a F5 NGINX Ingress Controller image with F5 NGINX App Protect WAF v5 from source code.
+
+{{<call-out "tip" "Pre-built image alternatives" >}} If you'd rather not build your own NGINX Ingress Controller image, see the [pre-built image options](#pre-built-images) at the end of this guide.{{</call-out>}}
+
+## Before you start
+
+- To use NGINX App Protect WAF with NGINX Ingress Controller, you must have NGINX Plus.
+
+## Prepare the environment
+
+Get your system ready for building and pushing the NGINX Ingress Controller image with NGINX App Protect WAF v5.
+
+1. Sign in to your private registry. Replace `<my-docker-registry>` with the path to your own private registry.
+
+    ```shell
+    docker login <my-docker-registry>
+    ```
+
+1. Pull the WAF Config Manager image:
+
+    ```shell
+    docker pull private-registry.nginx.com/nap/waf-config-mgr:<image-tag>
+    ```
+
+1. Pull the WAF Enforcer Docker image    
+
+    ```shell
+    docker pull private-registry.nginx.com/nap/waf-enforcer:<image-tag>
+    ```
+
+1. Clone the NGINX Ingress Controller repository:
+
+    ```console
+    git clone https://github.com/nginxinc/kubernetes-ingress.git --branch v3.6.0
+    cd kubernetes-ingress
```

Comment:
No idea what on earth is happening here. Does the formatting of the document break?

Reply:
fixed, updated to `shell`